### PR TITLE
Add CSV manifests for v0.4.1

### PR DESCRIPTION
**Pull request by lrgar**
Additionally,
* Includes the permissions required for Istio.
* Changing the OpenShift package name to "oneagent-certified" since that it's the one we use when publishing.
* Updating the Capabilities level to "Deep Insights".

### CODE REVIEW

File: deploy/olm/openshift/dynatrace.package.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-packageName: oneagent
+packageName: oneagent-certified
```
[the remark this answers is not in the document]
Comment (lrgar) [reply deploy/olm/openshift/dynatrace.package.yaml]: That is how the package eventually got named on [RedHat's OperatorHub](https://quay.io/application/certified-operators/oneagent-certified?tab=releases) and forgot to update it here.

**Comment by baichinger** [comment]
As discussed we will skip CSV for v0.4.0. Just create the CSV for v0.4.1 for the upcoming release.

File: deploy/olm/kubernetes/dynatrace-monitoring.v0.4.1.clusterserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+             "APP_LOG_CONTENT_ACCESS=1"
+          ],
+          "env": []
+       }
+      }]
+    capabilities: Deep Insights
+    categories: "Monitoring,Logging & Tracing,OpenShift Optional"
+    certified: "false"
+    containerImage: quay.io/dynatrace/dynatrace-oneagent-operator:v0.4.1
+    createdAt: 2019-02-06T12:59:59Z
```
Comment (baichinger) [inline deploy/olm/kubernetes/dynatrace-monitoring.v0.4.1.clusterserviceversion.yaml]: Should we update this field?

Comment (namratachaudhary) [reply deploy/olm/kubernetes/dynatrace-monitoring.v0.4.1.clusterserviceversion.yaml]: good catch, will do.

**Pull request merged**